src/components/editor/InputListLOC.jsx

```diff
@@ -50,13 +50,22 @@ class InputListLOC extends Component {
   }
 
   render() {
-    let lookupUri, isMandatory, isRepeatable
+    let isMandatory, isRepeatable
+
+    if (this.state.lookupUri === undefined) {
+      console.error("lookup URI was not defined!")
```

Comment:
in a perfect world, would this be an alert to the user?

Reply:
I added an alert.